docs/generated/raw/kuma-cp.yaml

```diff
@@ -825,6 +825,15 @@ ipam:
     cidr: 243.0.0.0/8 # ENV: KUMA_IPAM_MESH_MULTI_ZONE_SERVICE_CIDR
   # Interval on which Kuma will allocate new IPs for MeshServices and MeshExternalServices
   allocationInterval: 5s # ENV: KUMA_IPAM_ALLOCATION_INTERVAL
+  # KnownInternalCIDRs contains a list of CIDRs which are considered internal and trusted by Envoy when sending HTTP requests
+  knownInternalCIDRs: # ENV: KUMA_IPAM_ALLOCATION_INTERVAL
```

Comment:
Should we include our CIDRs for MeshService, MeshExternalService, and MeshMultiZoneService, as well as possibly 240.0.0.0? I'm not entirely sure when the address IP is validated, but if the outbound listener is bound to one of these IPs and validation happens at this step, we might miss some data.

Reply:
You mean to include these addresses automatically whenever they were set, right?
I think we can do it, yes

Reply:
Also, the environment var name is incorrect in this comment

Reply:
Yes, we could add them automatically.

Reply:
The `isInternalRequest` is used to check if a request is originated from an internal client, so it's not necessary to include these Mesh*Service CIDRs since they are only used as destinations.
https://github.com/envoyproxy/envoy/blob/main/source/common/http/conn_manager_utility.cc#L222-L251

Envoy attaches a set up request headers when forwarding requests to the upstream. It removes "internal only headers" from requests originated from external clients, so that, in this case, the upstream server only receives trusted/sanitized headers.